cp22/hpi-fun2/newsort/problem.tex

```diff
@@ -0,0 +1,28 @@
+\input{template.tex}
+
+\begin{document}
+
+\makeheader
+
+Rick and Morty are on a new adventure. This time they landed in a parallel universe and of course, Morty broke Ricks laser gun which is needed ot get back home. To get the money back for the materials for the gun they try to gamble at the stock market. Rick knows that it's a physical law, that the right stocks always go up - and thats where they want to invest. 
+
+You need to figure out if the values of a stock price you receive are increasing, so that Rick and Morty know where to invest. The problem is: In this parallel universe they use the same arabian digits but they don't have the same ordering. In our world we have the ordering $0 < 1 < 2 < 3 < 4 < 5 ...$ but in a parallel universe this ordering could be $6 < 2 < 1 < 4 \dots$ . This ordering does not apply to exponentials, so 11 is still bigger than 1. Luckily Rick was smart enough to already figure out the mapping hence he can give you the ordering of the digits from which you can read the new meaning of the number from the index in the list.
+
+Can you tell Rick and Morty if they should invest in a given stock?
+
+\paragraph*{Input}
+
+The first line contains the number of prices $1\leq n<10^5$ of a stock Rick picked for you. In the next line you receive the $n$ prices, whereas each price is a digit string with a maximum length of 9. These prices do not have leading "zeros", which may be represented in a new digit. In the last line you will receive the digit order of the local universe, wich is a permutation of the digits 0-9. The prices can have duplicate values.
+
+\paragraph*{Output}
+
+Output YES if the given stock prices are increasing and NO if not. It does not have to be strictly increasing.
+
+\begin{samples}
+  \sample{sample1}
+  \sample{sample2}
+  \sample{same_values}
```

Comment:
You can't know this yet: All testcases starting with `sample` will be available for download in the judge, all others are not. Thus, you should name all sample testcases starting with `sample`.